magicctapipe/lstchain/image/modifier.py

```diff
@@ -0,0 +1,138 @@
+import logging
+
+import numpy as np
+from numba import njit
+
+__all__ = [
+    "add_noise_in_pixels",
+    "random_psf_smearer",
+    "set_numba_seed",
+]
+
+log = logging.getLogger(__name__)
+
+
+# number of neighbors of completely surrounded pixels of hexagonal cameras:
+N_PIXEL_NEIGHBORS = 6
+SMEAR_PROBABILITIES = np.full(N_PIXEL_NEIGHBORS, 1 / N_PIXEL_NEIGHBORS)
+
+
+def add_noise_in_pixels(
```

Comment:
those functions also look the same (with some cosmetic differences) like in lstchain 0.10.6

Reply:
See comment on the lstchain __init__ file